src/main.rs

```diff
@@ -7,7 +7,7 @@ use url::Url;
 #[tokio::main]
 async fn main() {
     let client_config = Config {
-        url: Url::parse("https://fuel-beta-5.hypersync.xyz").unwrap(),
+        url: Url::parse("https://fuel-testnet-3.hypersync.xyz").unwrap(),
```

Comment:
@JossDuff does this need to change to drop the "-3" ? :)

Reply:
It will eventually, but it's fine for now.  I need to switch some ingresses around.  This is pointing to the HyperFuel deployment that has the fix for transaction types.